sentry_sdk/integrations/_wsgi_common.py

```diff
@@ -61,7 +62,7 @@ def extract_into_event(self, event):
         request_info = event.get("request", {})
 
         if _should_send_default_pii():
-            request_info["cookies"] = dict(self.cookies())
+            request_info["cookies"] = self.cookies()
```

Comment:
`self.cookies()` here should always already be a `dict` and there is no need to copy it twice now that we `deepcopy` the whole request a bit further down.